app/src/main/java/org/openimis/imispolicies/Renewal.java

```diff
@@ -186,6 +186,7 @@ protected void onCreate(Bundle savedInstanceState) {
                 }
 
                 if (!isValidate()) {
+                    pd = ProgressDialog.show(this, "", "");
                     pd.dismiss();
```

Comment:
I would remove both of these ProgressDialog references

Reply:
Not sure of what you meant